packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -158,8 +158,8 @@ export class ApiReferencesTable extends React.Component<IApiReferencesTableProps
     return this._isEnum ? (
       <ApiDetailsList itemKind="enum" items={properties as IApiEnumProperty[]} tokenResolver={tokenResolver} />
     ) : (
-      <ApiDetailsList itemKind="property" items={properties as IApiInterfaceProperty[]} tokenResolver={tokenResolver} />
-    );
+        <ApiDetailsList itemKind="property" items={properties as IApiInterfaceProperty[]} tokenResolver={tokenResolver} />
```

Comment:
Small thing but can you revert the spacing changes (here and later)? Otherwise they'll be reintroduced in someone else's PR when they merge with master since we run Prettier on commit.